From a4072e97fd98756ba6a0a878bdb876d782540819 Mon Sep 17 00:00:00 2001
From: Tri Luong <tri.luong.te@renesas.com>
Date: Fri, 5 Apr 2019 09:08:43 +0700
Subject: [PATCH 170/175] arm64: dts: r8a774c0-cat874: Define and enable
 Bluetooth pin.

This patch defines and enables pin for bluetooth device
used for RZ/GE board (R8A774C0 SoC).

Signed-off-by: Tri Luong <tri.luong.te@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts b/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
index afd965b..d005a59 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
@@ -327,3 +327,12 @@
 &vspi0 {
 	status = "okay";
 };
+
+&gpio4 {
+	bluetooth-en-gpio {
+		gpio-hog;
+		gpios = <6 GPIO_ACTIVE_HIGH>;
+		output-high;
+		line-name = "bluetooth-en-gpio";
+	};
+};
-- 
2.7.4

